test/common/router/config_impl_test.cc

```diff
@@ -965,8 +965,10 @@ TEST_F(RouteMatcherTest, TestRoutes) {
     EXPECT_EQ("www2", route->clusterName());
     EXPECT_EQ("www2", virtualHostName(route));
     route->finalizeRequestHeaders(headers, stream_info, true);
-    // TODO(samflattery): uncomment after https://github.com/google/re2/issues/272 resolved
-    /* EXPECT_EQ("/nXwforrXg2_Xndpoint/tXX?test=me", headers.get_(Http::Headers::get().Path)); */
+    // TODO(samflattery): remove after https://github.com/google/re2/issues/272 resolved
+#ifndef FUZZ_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
Can you test this out with `bazel run --config asan-fuzzer test/common/router:route_fuzz_test`?

Should be `FUZZING_BUILD_MODE...`

Reply:
fixed